README.md

```diff
@@ -917,6 +917,47 @@ We have [ticket #111](https://github.com/FormidableLabs/builder/issues/111) out
 to write a babel plugin to make the module pattern semantics available during
 babel transpilation as well.
 
+#### Frontend Resolution and Module Pattern
+
+An analogous situation occurs for frontend JS code in the production archetype,
+but with a different solution. If frontend web application code that has
+dependencies from a dev archetype,  the build environment will need to be
+enhanced to search within the dev archetype.
```

Comment:
I may not be understanding this correctly, but why should the application take a dependency from the _dev_ archetype?

Also grammar-🐐

> If frontend web application code ~~that~~ has dependencies